### PR TITLE
Add clarification to differentiate between the blog URL and the RSS(feed URL) during sign-up

### PR DESCRIPTION
**Pull request by cychu42**
## Issue This PR Addresses
USAGE: #3646 
This PR addresses one of the issues raised in the issue.

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

image
The issue was that some users experience confusion between a blog URL (blog's root HTML) and select the RSS (blog's feed XML) during sign-up. This PR adds 2 sentences to the prompt to clarify the difference, in `src\web\app\src\components\SignUp\Forms\BlogFeeds.tsx`. 

## Steps to test the PR

Test this by going through the sign-up process to view the Blog Feeds page.
During testing, you can use test account `user1` with password `user1pass`.
You can enter any Github account name to get through the page before this page.

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by NeilAn99** [comment]
I think it might be nice to provide an example of a blog URL, so users who don't know what HTML and XML are know what to provide.

**Comment by cychu42** [comment]
Maybe, but there are many different sites, so I'm not sure it's practical to list them all or useful to list only one.

**Comment by NeilAn99** [comment]
I think maybe one example would be fine, I think most people use dev.to. But just a simple example, something like:

Blog URL: https://dev.to/user
vs
Blog feed URL: https://dev.to/feed/user

**Comment by TueeNguyen** [comment]
Agree with @NeilAn99, let's add examples of blog url and feed url in 2 separate lines

**Comment by cychu42** [comment]
OK, how about this?
image

File: src/web/app/src/components/SignUp/Forms/RSSFeeds.tsx
Comment (batunpc) [inline src/web/app/src/components/SignUp/Forms/RSSFeeds.tsx:147]: Instead of `promptExample1` and `promptExample2`, won't it be better to take an array with prop example?

File: src/web/app/src/components/SignUp/Forms/ChannelFeeds.tsx
Comment (batunpc) [inline src/web/app/src/components/SignUp/Forms/ChannelFeeds.tsx:12]: Here making the prop optional and assigning a default value like empty list , ensure that nothing will break.

File: src/web/app/src/components/SignUp/Forms/RSSFeeds.tsx
Comment (batunpc) [inline src/web/app/src/components/SignUp/Forms/RSSFeeds.tsx:255]: Having an array will make these dynamic and reusable

**Review by batunpc — CHANGES_REQUESTED** [review]
@cychu42 Your implementation looks good!
I have left some initial review that we could address

**Comment by cychu42** [comment]
@batunpc Good ideas! I made some changes accordingly.
The only thing I didn't do is assigning a default value, because it doesn't look like the existing code does it for other variables of type `RSSFeedsFormProps` , but I did make the `promptExamples` array optional and make sure nothing would render if there's no value assigned. 
For instance, the next page renders using the same type without any example.
image

**Review by batunpc — APPROVED** [review]
Thank you @cychu42 , great job 👍

**Comment by cychu42** [comment]
Thank you all for the reviews. I just rebased and squashed the commits.
I guess I'm now waiting for a reviewer with write access to allow merge to happen.

**Review by batunpc — APPROVED** [review]

File: src/web/app/src/components/SignUp/Forms/RSSFeeds.tsx
Comment (manekenpix) [inline src/web/app/src/components/SignUp/Forms/RSSFeeds.tsx:256]:
```suggestion
              <div>{example}</div>
```

**Comment by manekenpix** [comment]
Last minute suggestion:

The text looks a bit bloated and hard to read, what about something more simple? maybe a basic but clear indication of what valid and non valid blog URLs are:

image

It doesn't have to be exactly this ^, but maybe something a bit cleaner would be better for users? It could also include links to blog URL vs feed URL explanations or definitions, or Blog vs Post (some people think a post is a blog).
I'm also ok if the team decides that the changes proposed in this PR are the way to go.

**Comment by cychu42** [comment]
What if we use some shorter text instead:
> Enter your blog root URL (HTML) and select the blog's RSS feed URL (XML) you want to use in Telescope, such as:
> Blog root URL: https://dev.to/user
> RSS feed URL: https://dev.to/feed/user

The 2 examples can be smaller texts like you suggested.

**Comment by manekenpix** [comment]
> What if we use some shorter text instead:
> 
> > Enter your blog root URL (HTML) and select the blog's RSS feed URL (XML) you want to use in Telescope, such as:
> > Blog root URL: https://dev.to/user
> > RSS feed URL: https://dev.to/feed/user
> 
> The 2 examples can be smaller texts like you suggested.

Yeah, I think that'd work.

File: src/web/app/src/components/SignUp/Forms/BlogFeeds.tsx
Comment (Myrfion) [inline src/web/app/src/components/SignUp/Forms/BlogFeeds.tsx:11]: Really picky thing, but I think you should add `the` before the blog
```suggestion
      A blog URL (HTML) should be the root of your blog itself, and Telescope will find the link(s) of the blog feed URL (XML) for you to select, such as:"
```

Comment (cychu42) [reply src/web/app/src/components/SignUp/Forms/BlogFeeds.tsx:11]: Thanks, but it won't apply after the new changes.

**Comment by cychu42** [comment]
OK. I updated it. It now looks like this:
image

For the prompt examples, I used the same font size as the `formControlLabel` class (the RSS URL after validation), which is smaller than the prompt.

**Review by manekenpix — APPROVED** [review]
LGTM :+1:

**Pull request merged**
